timon/plugins/http_server/views.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python
+
+# #############################################################################
+# Copyright : (C) 2023 by MHComm. All rights reserved
+#
+# __author__ = "Quentin Laymajoux"
+# __email__ = "info@mhcomm.fr"
+#
+# Name       : timon.plugins.http_server.views
+"""
+Summary: The http server plugin's views and routes
+"""
+# #############################################################################
+import json
+import logging
+import time
+
+from quart import request
+from quart_trio import QuartTrio
+
+from timon.probes.probe_if import mk_probe
+
+logger = logging.getLogger(__name__)
+
+app = QuartTrio(__name__)
+
+KNOWN_ROUTES = {
+    "/resources/": ("(GET) returns a list of all resources and their"
+                    " availability"),
+    "/queue/": "(GET) returns the heap as a list",
+    "/queue/lenght/": "(GET) returns the lenght of the heap",
+    "/queue/probe/<probename>/": "(GET) search probe in heap",
+    "/probes/<?probename>/run/": ("(GET) force run the probename and returns "
+                                  "the result"),
+    "/rescheduler/probes/": (
```

Comment:
what does the second level give us?

can you reschedule anything else than probes?

not usre, but perhaps in the future

/rescheduler/host ??